R/compile.R

```diff
@@ -17,12 +17,15 @@
 
 generate_rdefs <- function(pars,
                            cmt,
-                           func,
-                           init_fun="",
-                           table_fun="",
-                           config_fun="",
-                           model="",omats,smats,
-                           set=list(), 
+                           func = "",
+                           init_fun = "",
+                           table_fun = "",
+                           event_fun = "",
+                           config_fun = "",
+                           model = "",
+                           omats,
+                           smats,
+                           set = list(), 
```

Comment:
just light touchup; this code is really old and probably don't need to pass all the functions separately.